src/Text.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Text = ({ size = 'medium', color = '#000000', children }) => {
+  const sizes = {
+    small: '0.8rem',
+    medium: '1rem',
+    large: '1.2rem',
+  };
+
+  const sizeValue = sizes[size] || sizes.medium;
+
+  return (
+    <span style={{ fontSize: sizeValue, color }}>{children}</span>
```

Comment:
Just keep in mind that you may not always want to use `style` because using CSS classes is more performant.